scripts/postprocess-osx/packagebuilding-howto

```diff
@@ -1,7 +1,7 @@
 Steps tp create a OSX package:
 
 1) Build LibreCAD
-2) Test of LibreCAD is working, make sure resources are included like fonts, helpiles and templates
+2) Test of LibreCAD is working, make sure resources are included like fonts, helfiles and templates
```

Comment:
Thanks @lordofbikes 
I'll update this file again.
Do I need to delete this pull request and make new one using rebase or will it work fine if I use rebase without closing this pull request?